Node.js/basicresponse.js

```diff
@@ -156,9 +172,11 @@ function getMessage(hConn, hObj) {
       } catch (err) {
         debug_warn('JSON Parsing error ', err);
         debug_info("message <%s>", decoder.write(buf));
+        ok = false;
       }
     } else {
       debug_info("binary message: " + buf);
+      // ok = false;
```

Comment:
If its a binary message then this app can't handle it, so it must be returned to the queue. ie, Set ok to false.